src/tools/clippy/tests/ui/manual_map_option_2.stderr

```diff
@@ -21,16 +21,6 @@ LL +             (x, y.0)
 LL ~         });
    |
 
-error: manual implementation of `Option::map`
```

Comment:
What happened here?

Reply:
This is likely due to introduction of `Node::Let`, it can now be observed in a chain of `HirId` parents, and clippy could rely on it not being in the chain in some place.
I spent some limited time searching for that place, but couldn't find it.

I planned to make a clippy issue for this and for `src/tools/clippy/tests/ui/shadow.stderr` as well, because it also looks questionable (and related to `Node::Let`).